packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -459,7 +462,10 @@ function isOnboardingEnabled(provider: ProviderInfo, globalContext: ContextUI):
 
               {#if providerContainerConfiguration.has(provider.internalId)}
                 {@const providerConfiguration = providerContainerConfiguration.get(provider.internalId) || []}
-                <div class="flex mt-3 {container.status !== 'started' ? 'text-gray-900' : ''}">
+                <div
+                  class="flex mt-3 {container.status !== 'started' ? 'text-gray-900' : ''}"
+                  role="group"
+                  aria-label="ProviderConfiguration">
```

Comment:
maybe more human readable ?
```suggestion
                  aria-label="Configuration of the provider">
```

Reply:
`Provider configuration` is the same but shorter, right?